docs/API/index.md

```diff
@@ -50,6 +50,10 @@ And here you can see how it looks all in the User Interface:
 
 You can authenticate your requests by `X-AUTH-TOKEN` header, or by the query part of the request url: `?auth_token=YOUR_API_TOKEN`.
 
+An example query request with endpoint:
+
+`https://api.flotiq.com/api/v1/content/your_content_name?auth_token=YOUR_AUTH_TOKEN`:
```

Comment:
I think this would work better as a code example in multiple languages and if it had both examples for `X-AUTH-TOKEN` and `?auth_token` query param. See, e.g. [docs/API/content-type/updating-co.md](https://github.com/flotiq/flotiq-docs/blob/master/docs/API/content-type/updating-co.md) to get a sample for such code snippets.